keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -95,7 +95,7 @@ def __init__(
         self,
         classes,
         bounding_box_format,
-        backbone=None,
+        backbone="keras_cv.models>ResNet50V2Backbone",
```

Comment:
why the odd syntax ?

Reply:
+1 -- even if it means doing some string parsing or a dict lookup, I think it'd be nicer if the strings expected here don't include the fully-qualified namespace and the ">"

Reply:
Just want to echo that this shouldn't show up in the user facing API if possible

Reply:
Idea: why don't we use the preset name as a default? We already have a unique string for each model, so we can reuse and call `from_preset`!

We had this in KerasNLP for a while, but thought it was simpler to encourage each user to use `Task.from_preset` because having a default backbone was not important.

Reply:
I think a preset name makes sense here -- that idea sgtm